### PR TITLE
Add class-name translation to issues re BatchNormalizationV1 etc.

### PR DESCRIPTION
**Pull request by caisq**
Fixes https://github.com/tensorflow/tfjs/issues/1255

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-converter/334)

### CODE REVIEW

**Comment by caisq** [comment]
@pyu10055 The names have been updated. But as a comment in the newly added code stated, the bug is already out there in tf 1.13.1 and 1.0.0-alpha0. The fix is for the users who use those versions of tf.

File: python/tensorflowjs/converters/keras_h5_conversion.py [hunk 1]
```diff
@@ -127,6 +129,33 @@ def _ensure_json_dict(item):
...
+    'BatchNormalizationV1': 'BatchNormalization',
+    'UnifiedGRU': 'GRU',
+    'UnifiedLSTM': 'LSTM'
+}
+
+
+def translate_class_names(input_object):
+  if not isinstance(input_object, dict):
+    return input_object
+  out = copy.copy(input_object)
```
Comment (pyu10055) [inline python/tensorflowjs/converters/keras_h5_conversion.py]: can you replace the name inline instead of copying?

Comment (caisq) [reply python/tensorflowjs/converters/keras_h5_conversion.py]: I was thinking we should be save and not modify the input in place. But since the arg is not used elsewhere, modifying it in place should be find. I removed the copy.copy. Thanks.
PTAL

**Comment by RadEdje** [comment]
@caisq
Hi I'm using tf 2.0 alpha and tf.js 1.0.0
So I can use this code? Replace the existing code in my current installation and it should fix the BatchNormalizationV1 bug? Will it also change the next error? 

'''
Error: Improperly formatted model config for layer
'''

Or that requires fixing the serialiazation bug introduced in tf 1.13?


https://github.com/tensorflow/tfjs-converter/pull/334#issuecomment-472850614
